### PR TITLE
Fix erroneous matrix implementation

### PR DESCRIPTION
**Pull request by dbaynard**
Fixes erroneous 'matrix3d' where it should just be 'matrix'.

I also documented it.

### CODE REVIEW

File: src/Clay/Transform.hs [hunk 1]
```diff
@@ -114,8 +114,29 @@ skewY y = Transformation ("skewY(" <> value y <> ")")
...
+-- > matrix3d(a, b, 0, 0, c, d, 0, 0, 0, 0, 1, 0, tx, ty, 0, 1)
+matrix
+  :: Double -- ^ a (linear scale, x)
+  -> Double -- ^ b (skew, contribution of old y to new x)
+  -> Double -- ^ c (skew, contribution of old x to new y)
+  -> Double -- ^ d (linear scale, y)
+  -> Double -- ^ tx (translate x)
+  -> Double -- ^ ty (translate y)
+  -> Transformation
+matrix u v w x y z = Transformation ("matrix(" <> value [ u, v, w, x, y, z ] <> ")")
```
Comment (turion) [inline src/Clay/Transform.hs]: Thanks a lot for this fix! Can you call your variables in the docs the same way like in the function declaration? Or else, if you like the names in the docs better, fix the variables in the function declaration?

File: src/Clay/Transform.hs [hunk 2]
```diff
@@ -114,8 +114,29 @@ skewY y = Transformation ("skewY(" <> value y <> ")")
...
+-- \]
+--
+-- This means the new parameters are:
+--
+-- * @x: a x + c y + tx@
+-- * @y: b y + d y + ty@
+--
+-- It also corresponds to 
+-- 
+-- > matrix3d(a, b, 0, 0, c, d, 0, 0, 0, 0, 1, 0, tx, ty, 0, 1)
```
Comment (turion) [inline src/Clay/Transform.hs]: So should we maybe have a function `matrix3d` as well? If you agree and want to contribute that, feel free.

Comment (dbaynard) [reply src/Clay/Transform.hs:130]: I'm going to say no for now (scope creep).

File: src/Clay/Transform.hs (hunk 1 above)
Comment (dbaynard) [reply src/Clay/Transform.hs:139]: I've scheduled time to look at this next week. Thanks for your patience!

**Pull request closed without merging**